ironfish/src/rpc/routes/wallet/utils.ts

```diff
@@ -104,7 +107,33 @@ export function deserializeRpcAccountImport(accountImport: RpcAccountImport): Ac
           sequence: accountImport.createdAt.sequence,
         }
       : null,
+    multiSigKeys: accountImport.multiSigKeys
+      ? deserializeRpcAccountMultiSigKeys(accountImport.multiSigKeys)
+      : null,
+  }
+}
+
+export function deserializeRpcAccountMultiSigKeys(
+  rpcMultiSigKeys: RpcMultiSigKeys,
+): MultiSigKeys | null {
+  let multiSigKeys: MultiSigKeys | null = null
+
+  if (rpcMultiSigKeys) {
+    if (rpcMultiSigKeys.publicKeyPackage) {
+      multiSigKeys = { publicKeyPackage: rpcMultiSigKeys.publicKeyPackage }
```

Comment:
I would store other keys here too with `...rpcMultiSigKeys` in case coordinator wants to signer too in same account.

Reply:
@hughy, @patnir, and I briefly discussed this yesterday over huddle. I think it's reasonable to have the design such that the coordinator is not a signer. We can discuss more today if we'd like to support this or not.